examples/minimal/packages/client-vanilla/src/mud/getNetworkConfig.ts

```diff
@@ -13,7 +12,7 @@ type NetworkConfig = SetupContractConfig & {
 export async function getNetworkConfig(): Promise<NetworkConfig> {
   const params = new URLSearchParams(window.location.search);
 
-  const supportedChains = [localhost, latticeTestnet];
+  const supportedChains = [foundry, latticeTestnet];
```

Comment:
```suggestion
  const supportedChains: MudChain[] = [foundry, latticeTestnet];
```
to prevent typescript from complaining about `modeUrl` / `faucetUrl` missing on `foundry`